### ISSUE TITLE
Crashing

### ISSUE THREAD

**Issue by ThatGerex** [SaberAspect/SpectClient#5]
Hi fellow SaberAspect,
So i have somewhat of an issue, and that is that every time i try to click on a **module/hack or close the GUI**, the game just **crashes**...
I hope you read this message soon, because i think that this is a very good client and i don't want to miss it :D
Thank you.

**Comment by ThatGerex** [comment]
btw i am playing on v0.5.2

**Comment by SaberAspect** [comment]
Can you possibly my send the crash report that Minecraft is giving you?

**Comment by ThatGerex** [comment]
The game doesn't really seem to save a crash report file, although i can tell you what the registry writes..
[Crash.txt](https://github.com/SaberAspect/SpectClient/files/7533254/Crash.txt)
.

**Comment by ThatGerex** [comment]
Also, sorry for not seeing the msg, i was busy...

**Comment by SaberAspect** [comment]
Yeah, I looked at the crash report and it seems it’s something with the Mixins, is there any specific module you have toggled on? I’ll take a look later and if you can try and test different things

**Comment by SaberAspect** [comment]
Also tell me as specific as possible what excuse you do to make it crash

**Comment by SaberAspect** [comment]
The more specific the better

**Comment by ThatGerex** [comment]
No, i don't have any modules on, I just open my game and inmediately crashes :/

**Comment by ThatGerex** [comment]
And it says:
Oh no, your game crashed!
Process finished with code: 0

**Comment by ThatGerex** [comment]
Is there any way that i could help you with solving this? Do you want a screen recording to watch what happens?

**Comment by SaberAspect** [comment]
Oh shoot man I am caught up with work and stuff. I’ll try to help over the weekend, I’ll just send my dc, it’ll be better

**Comment by SaberAspect** [comment]
Saber#2141 That’s my discord

**Comment by ThatGerex** [comment]
I tried to add you, but the problem is that, your friend requests are closed...
My discord is                200_#6237

**Comment by master7720** [comment]
heres a fix dont use this client
